### PR TITLE
Added:To The Top Button

### LINKED ISSUE #287: Feature ✨: Add scroll to top button
Hey @SUGAM-ARORA ,

I have noticed a small issue that I believe could enhance the user experience. Currently, there is no scroll to top button present on the website, which makes navigation a bit cumbersome, especially when users need to return to the top of a lengthy page.
Though it may seem minor, the absence of this feature can impact the overall usability and performance of the website.

****Screenshot (Mandatory):****

image

@SUGAM-ARORA ,Could you please assign me this issue under gssoc'24 ?

Are you contributing under any open-source program ?
GSSOC'24

### PR DESCRIPTION
**Pull request by tkshsbcue**
added to the top button GSSOC

## Related Issue
Fixes #287 

## Description
Added To The Top Button

## Type of PR

- [x] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
So Basically we needed to add a button which takes you to the top of that Div so did that you can check

### CODE REVIEW

**Comment by tkshsbcue** [comment]
This is issues #287

**Review by SUGAM-ARORA — APPROVED** [review]
Approved
